android/build.gradle

```diff
@@ -16,8 +16,8 @@ apply plugin: 'com.android.library'
 apply from: 'https://gist.githubusercontent.com/hypest/742448b9588b3a0aa580a5e80ae95bdf/raw/8eb62d40ee7a5104d2fcaeff21ce6f29bd93b054/readReactNativeVersion.gradle'
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    compileSdkVersion 28
+    buildToolsVersion "28.0.3"
```

Comment:
Since we're touching this line, how about we completely remove the `buildToolsVersion` setting to let it get defined by the gradle plugin automatically?

Reply:
oh! 💯  agreed, done in a0461aa

Reply:
update: previous commit only affected the example app, this other commit does the job here 8cedb0c